src/utils/validationUtil.ts

```diff
@@ -569,4 +573,39 @@ export class ValidationUtil {
     //     throw new models.HTTPError(400, 'Failed IVA tests must have IVA Defects');
     //   }
   }
+
+  /**
+   * Validates central docs for an array of test types
+   * @param testTypes TestType[]
+   * @throws HTTPError with status 400 if validation fails
+   */
+  public static validateCentralDocs(testTypes: TestType[]): void {
+    testTypes.every((testType) => {
```

Comment:
Personally, I don't feel this is the correct use of `.every`. 

Typically, the use case of every would be to check every item in an array and then return the whether that satisfied some criteria, whereas this method returns void and disregarding the outcome of the every check.

I suggest changing to a `forEach` or `for of` loop.

Reply:
I did it this way as there are other functions here that use it in the same way and I wanted to keep it consistent e.g. ivaFailedHasRequiredFields

Reply:
Fair enough, in that case "ivaFailedHasRequiredFields" is also incorrectly using the every method.

Reply:
@Daniel-Searle - Looking at the code for "ivaFailedHasRequiredFields", this is not doing as you mentioned above. 

It was using the value returned from the every and then throwing `throw new models.HTTPError(400, 'Failed IVA tests must have IVA Defects');` - although that is now commented out, making the entire function redundant.

That is (if not commented out) a valid use of `.every`.